pkg/storage/stores/indexshipper/storage/util.go

```diff
@@ -82,7 +82,12 @@ func DownloadFileFromStorage(destination string, decompressFile bool, sync bool,
 		return err
 	}
 
-	level.Info(logger).Log("msg", "downloaded file", "total_time", time.Since(start))
+	if fStat, err := f.Stat(); fStat != nil && err == nil {
+		level.Info(logger).Log("msg", "downloaded file", "total_time", time.Since(start), "size", fStat.Size())
+	} else {
+		level.Info(logger).Log("msg", "downloaded file", "total_time", time.Since(start))
+	}
```

Comment:
How about doing it like this?
```suggestion
	fStat, err := f.Stat()
	if err == nil {
		logger = log.With(logger, "size", fStat.Size())
	}
```

Reply:
I like that. Thank you.